servers/collection-api/src/express.ts

```diff
@@ -63,6 +63,13 @@ export async function startServer(port: number): Promise<{
   app.use(
     adminUrl,
     cors<cors.CorsRequest>(),
+    express.json(),
```

Comment:
if the request body contains bad/invalid JSON, `express.json()` will throw and exception right? so the block below wouldn't be reached -- maybe worth considering adding an error handler?